services/data_plane_manager/src/main/resources/application.properties

```diff
@@ -19,7 +19,7 @@ protobuf.goal-state-message.version = 102
 #logging.level.root=INFO
 
 #####Ignite configuration######
-ignite.host=localhost
+ignite.host=172.16.50.5
```

Comment:
Please revert this change. You can keep this in your local branch but usually local environment setup parameters are not recommended to check in to common repository.